canisters/src/vts/src/lib.rs

```diff
@@ -1,7 +1,11 @@
-use candid::{CandidType, Deserialize};
+use candid::{CandidType, Deserialize, Principal};
 use ic_stable_structures::memory_manager::{MemoryId, MemoryManager, VirtualMemory};
 use ic_stable_structures::{DefaultMemoryImpl, StableBTreeMap};
 use std::cell::RefCell;
+use std::collections::HashMap;
```

Comment:
Let's use https://github.com/dfinity/stable-structures instead of basic ones. Our canister already have some.